db/test/test_data.py

```diff
@@ -118,6 +121,53 @@ def _get_ll_id_from_mbid(self, mbid):
                 ret.append(row[0])
             return ret
 
+    def test_load_many_low_level(self):
+        """Lowlevel data corresponds to the specified offsets, returns first submission without offset"""
+        one = {"data": "one", "metadata": {"audio_properties": {"lossless": True}, "version": {"essentia_build_sha": "x"}}}
+        two = {"data": "two", "metadata": {"audio_properties": {"lossless": True}, "version": {"essentia_build_sha": "x"}}}
+        three = {"data": "three", "metadata": {"audio_properties": {"lossless": True}, "version": {"essentia_build_sha": "x"}}}
+        db.data.write_low_level(self.test_mbid, one, gid_types.GID_TYPE_MBID)
+        db.data.write_low_level(self.test_mbid, two, gid_types.GID_TYPE_MBID)
+        db.data.write_low_level(self.test_mbid_two, three, gid_types.GID_TYPE_MBID)
+        
+        recordings = [(self.test_mbid, 0),
+                      (self.test_mbid, 1),
+                      (self.test_mbid_two, 0)]
+
+        ll_expected = {
+            self.test_mbid: {'0': one, '1': two},
+            self.test_mbid_two: {'0': three}
+        }
+        
+        self.assertEqual(ll_expected, db.data.load_many_low_level(list(recordings)))
+
+    def test_load_many_low_level_none(self):
+        """Exception is raised if no lowlevel data is loaded. 
+           If offset is too high or there are no submissions for the mbid, it is skipped.""" 
+
+        # No submitted lowlevel data for any mbid, and offset combinations
+        recordings = [('4ee71816-e0be-4257-a5f9-98dca3ec8bcd', 0),
+                      ('48877286-42d4-4b0a-a1e0-d703a587f64b', 1),
+                      ('ffcc4249-28bb-4c91-9195-a60b21d4fb94', 0)]
+        with self.assertRaises(db.exceptions.NoDataFoundException):
+            db.data.load_many_low_level(list(recordings))
```

Comment:
Is this what we talked about in IRC? Even if there are _no_ matches to the queried mbids/offsets I would expect to get an empty dictionary response, rather an an error

Reply:
Sorry, raising the error for no matches was an oversight. I've fixed it to use an empty dictionary and updated the tests in the most recent commit 5c6b601.